src/main/java/io/github/dsheirer/record/wave/AudioPacketWaveRecorder.java

```diff
@@ -110,7 +111,7 @@ public void start()
                 sb.append(TimeStamp.getLongTimeStamp("_"));
                 sb.append(".tmp");
 
-                mFile = Paths.get(sb.toString());
+                mFile = Paths.get(StringUtils.replaceIllegalCharacters(sb.toString()));
```

Comment:
This can be removed with the changes from review comment #2 so that linux file paths won't get corrupted.